### PR TITLE
renamed fire-and-rescue-service to fire-authority and removed unecces…

### PR DESCRIPTION
**Pull request by Jamie-Atkinson**
…dary fields from the register

can you both review please

### CODE REVIEW

File: data/discovery/register/fire-authority.yaml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+fields:
+- fire-authority
+- name
+- start-date
+- end-date
+phase: discovery
+register: fire-authority
+registry: home-office
+text: A register of fire authoritites in the England
```
Comment (nacnudus) [inline data/discovery/register/fire-authority.yaml]: `A register of fire authorities in England`

Comment (Jamie-Atkinson) [reply data/discovery/register/fire-authority.yaml]: i knew that :P

File: data/discovery/register/fire-authority.yaml [hunk 2]
```diff
@@ -6,4 +6,4 @@ fields:
 phase: discovery
 register: fire-authority
 registry: home-office
-text: A register of fire authoritites in the England
+text: A register of fire authorities in the England
```
Comment (nacnudus) [inline data/discovery/register/fire-authority.yaml]: `A register of fire authorities in England` ;)

File: data/discovery/register/fire-authority.yaml [hunk 3]
```diff
@@ -0,0 +1,9 @@
+fields:
+- fire-authority
+- name
+- start-date
+- end-date
+phase: discovery
+register: fire-authority
+registry: home-office
+text: A register of fire authorities in the England
```
Comment (nacnudus) [inline data/discovery/register/fire-authority.yaml]: `A register of fire authorities in England` ;)

**Pull request merged**